src/Components/TimelinePost/TimelinePost.js

```diff
@@ -2,16 +2,22 @@ import RecommendeeCard from "../RecommendeeCard/RecommendeeCard"
 import RecommendeeIcon from "../RecommendeeIcon/RecommendeeIcon"
 import './_TimelinePost.scss'
 
-const TimelinePost = () => {
+const TimelinePost = ({ recommenderAvatar, recommenderName, poster, title, releaseYear, rating, genres, date }) => {
     return (
         <div className="timeline-post-container">
-            <RecommendeeIcon />
+            <RecommendeeIcon recommenderAvatar={recommenderAvatar} recommenderName={recommenderName}/>
             <div className="timeline-segment">
-            <div className="timeline-tail"></div>
-            <div className="timeline-circle"></div>
-            <div className="timeline-tail"></div>
+                {/* <div className="timeline-tail"></div> */}
+                <div className="timeline-circle"></div>
             </div>
-            <RecommendeeCard />
+            <RecommendeeCard
```

Comment:
Same global context and API question here. Could we just use useContext here instead of passing props?